src/server/index.test.ts

```diff
@@ -469,3 +471,50 @@ describe('Model cache', () => {
     })
   })
 })
+
+describe('Base pages', () => {
+  let server: Server
```

Comment:
I'll move these into our Routes tests file **src/server/routes/index.test.ts**

Cheers for adding them